### DISCUSSION TITLE
Get city's details from API

### DISCUSSION THREAD

**Discussion by halbtale** [dr5hn/countries-states-cities-database#258] [Q&A] [answered]
Hi!
Is there a way to get city's details from the API "https://countrystatecity.in/"?
It seems that you can get a list of cities by country, but it will only return name and id. 
How can I get other city's details like latitude and longitude?

**Comment by dr5hn** [comment, marked as answer]
@Nenino2 It's currently not implemented at the moment. 

But good to have, Will surely implement a route to get the additional city details.
No ETA, please.
